src/internal/tester/resource_owners.go

```diff
@@ -42,8 +55,29 @@ func LoadTestM365UserID(t *testing.T) string {
 	return cfg[TestCfgLoadTestUserID]
 }
 
-// expects cfg value to be a string representing an array like:
-// "['foo@example.com','bar@example.com']"
+// expects cfg value to be a string representing an array such as:
+// ["site1\,uuid","site2\,uuid"]
+// the delimeter must be a |.
```

Comment:
this comment does not seem to match the array given on the line above

Reply:
Yeah... we have some things to figure out about resource owner lookup with sites.  I can come back and correct his when we do.